bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngWig",
-  "version": "3.0.0-rc1",
+  "version": "3.0.1",
```

Comment:
Fully agree. Actually that commit was by mistake, didn't know that grunt:build would do a version bump. Only took a closer look at the gruntfile after.